modules/photon-cdn.php

```diff
@@ -31,6 +31,7 @@ public static function go() {
 		add_action( 'admin_print_styles', array( __CLASS__, 'cdnize_assets' ) );
 		add_action( 'wp_footer', array( __CLASS__, 'cdnize_assets' ) );
 		add_filter( 'load_script_textdomain_relative_path', array( __CLASS__, 'fix_script_relative_path' ), 10, 2 );
+		Jetpack::enable_module_configurable( __FILE__ );
```

Comment:
Is this necessary since the feature has no configuration settings, just an on/off toggle?

Reply:
Nope. Can be removed.

Reply:
This was left in accidentally, after we used it for testing. Removed in 539b320